### PR TITLE
feat(postman): add test cases for payouts

### PR DESCRIPTION
**Pull request by kashif-m**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
https://github.com/juspay/hyperswitch/issues/3978

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?
**Postman**
image

**cargo test-utils using newman**
image

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: crates/test_utils/src/connector_auth.rs [hunk 1]
```diff
@@ -93,7 +93,7 @@ impl ConnectorAuthentication {
 }
 
 #[derive(Clone, Debug, Deserialize)]
-pub struct ConnectorAuthenticationMap(HashMap<String, ConnectorAuthType>);
+pub struct ConnectorAuthenticationMap(HashMap<String, CollectionAuthType>);
```
Comment (pixincreate) [inline crates/test_utils/src/connector_auth.rs:96]: Please revert these changes back, it has many dependencies and I'm pretty sure it will break UI and Webhook UI tests.

File: crates/test_utils/src/connector_auth.rs [hunk 2]
```diff
@@ -159,33 +159,28 @@ impl ConnectorAuthenticationMap {
                             table.get("key1"),
                             table.get("api_secret"),
                             table.get("key2"),
+                            table.get("adyen"),
+                            table.get("wise"),
```
Comment (pixincreate) [inline crates/test_utils/src/connector_auth.rs:163]: Can we please keep the code as connector agnostic as possible and make as much changes as possible within the collection itself.

Ideally, we would want to modify this only if we want to add yet another key type that takes 5 more keys.

File: crates/test_utils/src/connector_auth.rs [hunk 3]
```diff
@@ -326,6 +376,10 @@ pub enum ConnectorAuthType {
         api_secret: Secret<String>,
         key2: Secret<String>,
     },
+    PayoutKeys {
+        adyen: SignatureKey,
+        wise: BodyKey,
+    },
```
Comment (pixincreate) [inline crates/test_utils/src/connector_auth.rs:382]: I think we could achieve this by tweaking around the `pre_request.js` script in MCA step. Something like:
```txt
|-MCA_dir
    |- MCA_adyen
    \- MCA_wise
```

pass the auth for both in a single go, and then, extract?

Also, Adyen and Wise are 2 different connectors, can we have 2 different collections by separating them? This will greatly reduce complexity from what I understand

File: postman/collection-dir/payout/#1 Connector integration sanity/Adyen/Create Card Payout/request.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
+{
+  "method": "POST",
+  "header": [
+    {
+      "key": "x-feature",
+      "value": "integ-custom",
+      "type": "text"
+    }
```
Comment (pixincreate) [inline postman/collection-dir/payout/#1 Connector integration sanity/Adyen/Create Card Payout/request.json:8]: Remove, everywhere if present

File: postman/collection-dir/payout/#1 Connector integration sanity/Adyen/Create SEPA Payout/request.json [hunk 5]
```diff
@@ -0,0 +1,75 @@
+{
+  "method": "POST",
+  "header": [
+    {
+      "key": "x-feature",
+      "value": "integ-custom",
+      "type": "text"
+    }
```
Comment (pixincreate) [inline postman/collection-dir/payout/#1 Connector integration sanity/Adyen/Create SEPA Payout/request.json:8]: remove

File: postman/collection-dir/payout/#1 Connector integration sanity/Wise/Create SEPA Payout/request.json [hunk 6]
```diff
@@ -0,0 +1,75 @@
+{
+  "method": "POST",
+  "header": [
+    {
+      "key": "x-feature",
+      "value": "integ-custom",
+      "type": "text"
+    }
```
Comment (pixincreate) [inline postman/collection-dir/payout/#1 Connector integration sanity/Wise/Create SEPA Payout/request.json:8]: here as well

File: postman/collection-dir/payout/#2 Payout in stages - create + confirm + fulfill API calls/Bank payouts/Create SEPA Payout/request.json [hunk 7]
```diff
@@ -0,0 +1,75 @@
+{
+  "method": "POST",
+  "header": [
+    {
+      "key": "x-feature",
+      "value": "integ-custom",
+      "type": "text"
+    }
```
Comment (pixincreate) [inline postman/collection-dir/payout/#2 Payout in stages - create + confirm + fulfill API calls/Bank payouts/Create SEPA Payout/request.json:8]: here as well. we've this part code in many places down the line, please look after it.

**Comment by kashif-m** [comment]
not needed
